include/seqan3/core/bit_manipulation.hpp

```diff
@@ -15,6 +15,10 @@
 
 #include <meta/meta.hpp>
 
+#ifndef __GNUC__
```

Comment:
Why this guard?

Reply:
Ah you say if it is not defined and then you use a fallback that is implemented in the sdsl. I see.